### PR TITLE
Improve MyScale Hybrid Search and Add `DELETE` for MyScale vector store

### PR DESCRIPTION
**Pull request by mpskex**
# Description

This change is made by @suchme19.

-  add the delete func of myscale vectorstore  and update the corresponding document
-  adjust the pre search limit ratio about hybrid search to avoid excessively large pre search result

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [X] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Comment by nerdai** [comment]
Thanks @mpskex. I noticed that lint check has failed in our CI. Can you please run `make lint`?

**Review by nerdai — CHANGES_REQUESTED** [review]
Thanks for contributing to this @mpskex! 🔥 

Just some minor things.

**Comment by nerdai** [comment]
@mpskex One other thing: can you please update the `CHANGELOG.md` with a new line of a few words describing this new feature add?

File: docs/examples/vector_stores/MyScaleIndexDemo.ipynb [hunk 1]
```diff
@@ -79,16 +79,7 @@
...
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Document ID: a5f2737c-ed18-4e5d-ab9a-75955edb816d\n",
-      "Number of Documents:  1\n"
-     ]
-    }
-   ],
```
[the remark this answers is not in the document]
Comment (mpskex) [reply docs/examples/vector_stores/MyScaleIndexDemo.ipynb]: Reverted

File: docs/examples/vector_stores/MyScaleIndexDemo.ipynb [hunk 2]
```diff
@@ -109,15 +100,7 @@
...
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "../data/paul_graham/paul_graham_essay.txt\n"
-     ]
-    }
-   ],
```
[the remark this answers is not in the document]
Comment (mpskex) [reply docs/examples/vector_stores/MyScaleIndexDemo.ipynb]: Kept

**Comment by mpskex** [comment]
> @mpskex One other thing: can you please update the `CHANGELOG.md` with a new line of a few words describing this new feature add?

Added into new features and bug fixes

**Review by nerdai — APPROVED** [review]
LGTM!!

Thank you.

**Comment by nerdai** [comment]
Merging this now.

**Pull request merged**